.travis.yml

```diff
@@ -20,13 +20,14 @@ matrix:
   include:
   - os: osx
     language: generic
-    env: NMODE=nikola
+    env: NMODE=nikola OS=macOS
```

Comment:
They’re hard to spot, especially if you’ve never looked there before (and the icons are barely-visible grey)